cli/bpmetadata/tfconfig.go

```diff
@@ -415,28 +428,60 @@ func parseBlueprintRoles(rolesFile *hcl.File) ([]*BlueprintRoles, error) {
 			return nil, err
 		}
 
-		for k := range iamAttrs {
-			var iamRoles []string
-			attrValue, _ := iamAttrs[k].Expr.Value(nil)
-			if !attrValue.Type().IsTupleType() {
-				continue
+		// Special branch for perModuleMode
+		if perModuleMode {
```

Comment:
What would happen if someone runs in perModuleMode on an existing terraform blueprint which has roles in older format.

Reply:
there should be some quick checks, for example the `per_module_roles` local need to be defined. I'll update accodingly.